src/screens/reader/components/TextReader.tsx

```diff
@@ -82,7 +82,7 @@ const TextReader: React.FC<TextReaderProps> = ({
               title={`Next: ${nextChapter.chapterName}`}
               onPress={navigateToNextChapter}
               theme={theme}
-              textColor={theme.colorButtonText}
+              textColor={theme.colorButtonText} // what is this prop ?
```

Comment:
To override the button's text color.

Reply:
but I didnt see colorButtonText in that interface

Reply:
yeah, this was removed when themes were migrated to Material You, and I missed removing this.